spec/spec_helper.rb

```diff
@@ -94,4 +94,19 @@
   # test failures related to randomization by passing the same `--seed` value
   # as the one that triggered the failure.
   Kernel.srand config.seed
+
+  config.before(:context) do
+    # Capture the current time so that we can compare against files afterward
+    # to clear out any downloaded/exported plans/templates
+    @start_time = Time.now
+  end
+
+  config.after(:context) do
+    # Clean up any files generated by the Export/Download UI pages
+    path = Rails.root.join("*.{csv,txt,docx,pdf}").to_s
+    Dir.glob(path).each do |file|
+      puts "Deleting test file generated by Download/Export: #{file}"
+      File.delete(file) if File.ctime(file) > @start_time
+    end
```

Comment:
Thank you!
no more running `rm *.pdf *.docx *.txt *.csv` after testing:)

Reply:
exactly. I had accidentally run that command with a space after the asterisk and obliterated everything in the root dir!